fluent-syntax/src/parser/pattern.rs

```diff
@@ -174,8 +174,8 @@ where
                 b'\r' if self.is_byte_at(b'\n', self.ptr + 1) => {
                     self.ptr += 1;
                     return Ok((
-                        start_pos,
-                        self.ptr - 1,
+                        start_pos + 1,
```

Comment:
Question: Is this change covered by a test?

Thought: I'm guessing it will be covered by the roundtripping tests, and has coverage there. This change makes me a little nervous compared to the rest of the PR since it's changing behavior in another crate.

Reply:
Yes, it is covered by the fixture crlf.ftl which doesn't roundtrip correctly without it.
I could write a unit test in parser/pattern.rs to demonstrate the issue. But as the intended behaviour of that code is not generally comprehensible I'm not sure what this would accomplish.